### DISCUSSION TITLE
VS Code and Copilot version for Windows 7

### DISCUSSION THREAD

**Discussion by rjsdotorg** [community/community#137302] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

For project reasons, this 2011 i7 machine is sticking to Win7.
VSCode 1.70.3 runs fine, but no Copilot version I can find will respond. No errors, just no function, code completion etc.
Tried multiple sign-in/outs and have a paid Github.

The zip versions of VS Code up to at least 1.79.2 do run on Win7, and Copilot version GitHub Copilot  v1.90.174 seems to work - ie code completion.
**_Q: what is the latest .zip distro of VSC that will run with Win7, and the commensurate Copilot version?_**

VSC 1.86.0 does NOT run on Win7 (a kernel issue of some sort I recall).
I did spend quite some time looking for a list of VSC<>Copilot to no avail.

**Comment by mohitpaddhariya** [comment]
# VS Code and GitHub Copilot Setup for Windows 7

## Overview
This document provides the details for the latest compatible versions of Visual Studio Code (VS Code) and GitHub Copilot that work on Windows 7.

## VS Code Version
- **Latest Compatible Version**: 1.79.2
- **Download URL**: [VS Code 1.79.2](https://code.visualstudio.com/updates/v1_79)

## GitHub Copilot Extension
- **Compatible Version**: v1.90.174
- **Download URL**: [GitHub Copilot v1.90.174](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot)

## Installation Instructions
1. **Install VS Code**:
   - Download the zip file for VS Code version 1.79.2.
   - Extract the contents to your preferred directory.
   - Run `Code.exe` to start VS Code.

2. **Install GitHub Copilot**:
   - Open VS Code.
   - Go to Extensions (Ctrl+Shift+X).
   - Search for "GitHub Copilot".
   - Install version v1.90.174.

## Troubleshooting
- If you encounter issues with the GitHub Copilot extension not functioning, ensure that you are using the correct version of both VS Code and the extension as listed above.
- Make sure that your GitHub account is properly signed in to VS Code.

## Additional Notes
- Newer versions of VS Code and GitHub Copilot may not be compatible with Windows 7 due to updated dependencies and system requirements.

## References
- [Visual Studio Code Release Notes](https://code.visualstudio.com/updates)
- [GitHub Copilot Extension](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot)

**Reply by LiteBrite82** [reply, marked as answer]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
